src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const operations = {
```

Comment:
What  does you mean, when you called an object as a 'operations', I think, it's better, if you called this one as a 'calculator'.

Reply:
> What does you mean, when you called an object as a 'operations', I think, it's better, if you called this one as a 'calculator'.

In the name of the function, it is quite clear that this is a calculator

So in an object, I use the name "operations", because what happens in the function is exactly operations or "actions"

Add, subtract, and so on... ;)